File: PISSolution.Server/Models/Property.cs
namespace PISSolution.Models
{
    public class Property
    {
       
            public Guid ID { get; set; }
            public string PropertyName { get; set; }
            public string Address { get; set; }
            public decimal Price { get; set; }
            public DateTime DateOfRegistration { get; set; }

            public ICollection<Ownership>? Ownerships { get; set; } 
            public ICollection<PriceHistory>? PriceHistories { get; set; }
       
        
    }
}

File: pissolution.client/src/app/model/property.ts
import { PriceHistory } from "./pricehistory";
import { Ownership } from "./ownership";

export interface Property {
  id: string;
  propertyName: string;
  address: string;
  price: number;
  dateOfRegistration: Date;
  ownerships?: Ownership[];
  priceHistories?: PriceHistory[];
}
